### PR TITLE
Changed ClientHost and DistroHost in configuration to be consistent w…

### PR DESCRIPTION
**Pull request by dylanhantula**
…ith other services

Signed-off-by: Dylan Hantula <dylanhantula@gmail.com>

### CODE REVIEW

File: cmd/export-client/res/configuration.toml [hunk 1]
```diff
@@ -13,6 +13,6 @@ ConsulHost = 'localhost'
 ConsulPort = 8500
 CheckInterval = '10s'
 ConsulProfilesActive = 'go'
-ClientHost = 'EXPORT_CLIENT_HOST'
-DistroHost = 'EXPORT_CLIENT_DISTRO_HOST'
+ClientHost = 'edgex-export-client'
```
Comment (feclare) [inline cmd/export-client/res/configuration.toml]: Change to localhost, as the default configuration file assumes that all services are running in the same machine.

File: cmd/export-client/res/configuration-docker.toml [hunk 2]
```diff
@@ -13,6 +13,6 @@ ConsulHost = 'edgex-core-consul'
 ConsulPort = 8500
 CheckInterval = '10s'
 ConsulProfilesActive = 'docker;go'
-ClientHost = 'EXPORT_CLIENT_HOST'
-DistroHost = 'EXPORT_CLIENT_DISTRO_HOST'
+ClientHost = 'edgex-export-client'
```
Comment (jduranf) [inline cmd/export-client/res/configuration-docker.toml]: ClientHost is not used in export-client. Should be better to remove it

File: cmd/export-client/res/configuration-docker.toml [hunk 3]
```diff
@@ -13,6 +13,5 @@ ConsulHost = 'edgex-core-consul'
 ConsulPort = 8500
 CheckInterval = '10s'
 ConsulProfilesActive = 'docker;go'
-ClientHost = 'edgex-export-client'
```
Comment (jduranf) [inline cmd/export-client/res/configuration-docker.toml]: ClientHost also should be removed in export/client/const.go

**Comment by JPWKU** [comment]
recheck

**Comment by feclare** [comment]
recheck

**Comment by chenchix** [comment]
recheck

**Comment by chenchix** [comment]
recheck

**Pull request merged**